ipc/cli/src/commands/checkpoint/relayer.rs

```diff
@@ -95,4 +116,16 @@ pub(crate) struct BottomUpRelayerArgs {
         help = "The max parallelism for submitting checkpoints"
     )]
     pub max_parallelism: usize,
+    #[arg(
+        long,
+        default_value = "true",
+        help = "Flag to turn on/off Prometheus metrics reporting."
+    )]
+    pub enable_metrics: bool,
```

Comment:
```suggestion
    pub metrics_enabled: bool,
```

I reckon `--metrics-enabled` and `--metrics-listen-addr` rhymes better. Plus we use `enabled` as a nested property in the fendermint configs.